server/views/pages/activities/activitiesMigration.njk

```diff
@@ -97,9 +97,11 @@
                           </div>
                         </div>
 
-                        {% if u.whenEnded %}
-                          <div class="govuk-grid-row govuk-!-margin-bottom-2">
-                          {% if u.recordsFailed + u.recordsMigrated < u.estimatedRecordCount %}
+                        <div class="govuk-grid-row govuk-!-margin-bottom-2">
+                          <div class="govuk-grid-column-full">
+                            <a data-qa="all-events-link" class="govuk-link"  target="_blank" href="{{ u.appInsightsFullMigrationLink }}">View all migration events in App Insights</a>
+                          </div>
+                          {% if (u.whenEnded) and (u.recordsFailed + u.recordsMigrated < u.estimatedRecordCount) %}
```

Comment:
Changed the app insights query links visibility such that:

- always show the "all migration events" query, even when migration in progress
- only show the ignored migrations query when finished (we don't know if there are any ignored until finished)
- always show the failures query if we have failures, even when migration in progress